pkg/storage/v2/mysql/client.go

```diff
@@ -38,15 +40,22 @@ type options struct {
 
 func defaultOptions() *options {
 	return &options{
-		connMaxIdleTime: 300 * time.Second,
+		connMaxLifetime: 10 * time.Minute,
```

Comment:
Because the `connMaxIdleTime` is supposed to be less than the `connMaxLifetime`, I implemented the default value to 10 minutes.